File: src/ZohoDocsSDK/ZohoDocsSDK/Interfaces/IItems.vb
Public Interface IItems

    ''' <summary>
    ''' Moves multiple files/folders to a specified location
    ''' </summary>
    ''' <param name="DestinationFolderID">Mandatory. Id of folder to copy the new files/folders</param>
    Function FD_Move(DestinationFolderID As String) As Task(Of Boolean)
    ''' <summary>
    ''' Copies multiple files to a new location
    ''' https://apidocs.zoho.com/files/v1/copy?authtoken=AuthToken&scope=docsapi
    ''' </summary>
    ''' <param name="DestinationFolderID">Mandatory. Id of folder to copy the new files</param>
    Function F_Copy(DestinationFolderID As String) As Task(Of Boolean)
    ''' <summary>
    ''' Copies the existing folders to a specified folder
    ''' https://apidocs.zoho.com/files/v1/folders/copy?authtoken=AuthToken&scope=docsapi
    ''' </summary>
    ''' <param name="DestinationFolderID">Mandatory. Id of folder to copy the new folders</param>
    Function D_Copy(DestinationFolderID As String) As Task(Of Boolean)

End Interface

File: src/ZohoDocsSDK/ZohoDocsSDK/Interfaces/IItems.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Threading.Tasks;

namespace ZohoDocsSDK
{
  public   interface IItems
    {
        /// <summary>
        ///Moves multiple files/folders to a specified location
        ///</summary>
        ///<param name="DestinationFolderID">Mandatory. Id of folder to copy the new files/folders</param>
        Task<bool> FD_Move(string DestinationFolderID);
        /// <summary>
        ///Copies multiple files to a new location
        ///https://apidocs.zoho.com/files/v1/copy?authtoken=AuthToken&scope=docsapi
        ///</summary>
        ///<param name="DestinationFolderID">Mandatory. Id of folder to copy the new files</param>
        Task<bool> F_Copy(string DestinationFolderID);
        /// <summary>
        ///Copies the existing folders to a specified folder
        ///https://apidocs.zoho.com/files/v1/folders/copy?authtoken=AuthToken&scope=docsapi
        ///</summary>
        ///<param name="DestinationFolderID">Mandatory. Id of folder to copy the new folders</param>
        Task<bool> D_Copy(string DestinationFolderID);
    }
}
